Marathi/Shivaji-Maharaj/राजे शिवाजी महाराज.md

```diff
@@ -0,0 +1,17 @@
+छत्रपती शिवाजीराजे भोसले (१९ फेब्रुवारी १६३० ते ३ एप्रिल १६८०) हे इ.स. १८१८ पर्यंत टिकलेल्या आणि आपल्या परमोत्कर्षाच्या अवस्थेत भारतीय उपखंडाचा बराचसा भाग व्यापणार्‍या मराठा साम्राज्याचे संस्थापक होते.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.